### PR TITLE
SBP-2123 | Ensures that Instagram posts with no comments do not throw NP...

### PR DESCRIPTION
**Pull request by robdouglas**
...Es

### CODE REVIEW

File: streams-contrib/streams-provider-instagram/src/test/java/org/apache/streams/instagram/serializer/util/InstagramActivityUtilTest.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        Activity activity = new Activity();
+
+        Comments comments = new Comments();
+        comments.setComments(null);
+
+        data.setComments(comments);
+
+        InstagramActivityUtil.addInstagramExtensions(activity, data);
+
+        assert(activity != null);
```
Comment (sieverssj) [inline streams-contrib/streams-provider-instagram/src/test/java/org/apache/streams/instagram/serializer/util/InstagramActivityUtilTest.java]: This looks like a native assert().  Please us assertThat() or assertTrue() or the like.

**Comment by robdouglas** [comment]
Responded to code review feedback

**Comment by sieverssj** [comment]
:+1:

**Comment by smashew** [comment]
Just saw this will respond to code feedback when I get in

**Pull request merged**